{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/views/public.py

```diff
@@ -36,4 +38,12 @@ def get(self):
         # something like Cronitor is hitting this URL repeatedly to monitor uptime.
         log.info('ping-db ok')
 
-        return '{} ok'.format(flask.current_app.name)
+        end = ctasks.ping.apply_async().wait()
```

Comment:
-1 on .wait() here, let Cronitor do it's job IMO.

Reply:
Remove.